resources/uk/gov/hmcts/flux/reconcile-flux-image-repository.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# set -x
+
+CHART_DIRECTORY=${1}-${2}
+# CHART_DIRECTORY="plum-frontend"
+
+if [[ $CHART_DIRECTORY = "plum-frontend" ]]; then
+    CHART_DIRECTORY="cnp-plum-frontend"
+fi
+
+echo "Checking for flux image repository"
+
+env AZURE_CONFIG_DIR=/opt/jenkins/.azure-jenkins az login --identity > /dev/null
+
+env AZURE_CONFIG_DIR=/opt/jenkins/.azure-jenkins az aks get-credentials --resource-group ${3} --name ${4} --subscription ${5} -a > /dev/null 
```

Comment:
I would just set this before starting the script, to make it easier for someone else to run this locally for testing